Woof/Woof/Model/Sitter/Sitter+Dummy.swift

```diff
@@ -20,5 +20,20 @@ extension Sitter {
             rating: 4.8,
             pricePerHour: 40.25
         )
+
+        /// The instance of dummy sitter list to use in preview of views.
+        static let sitters = [
+            // swiftlint:disable line_length
```

Comment:
Nit:
Maybe we should remove the comments?

Reply:
I needed this line to disable the linter for line length violation. It was refactored, and we don't need it now.